wolfcrypt/src/port/Espressif/esp_crt_bundle/esp_crt_bundle.c

```diff
@@ -0,0 +1,1380 @@
+                            s_crt_bundle.num_certs);
+            ret = WOLFSSL_SUCCESS;
+        }
+    }
+
+    /* Get the current cert from the store. */
+    if (ret == WOLFSSL_SUCCESS) {
+        /* Get the current certificate being verified during the certificate
+         * chain validation process. */
+#ifdef OPENSSL_EXTRA
+    #ifdef WOLFSSL_ALT_CERT_CHAINS
+            /*  Retrieve the last WOLFSSL_BUFFER_INFO struct with alt chains */
+            buffer = store->certs[store->totalCerts - 1];
+            store_cert = wolfSSL_X509_d2i(NULL,
+                                          (const unsigned char*)buffer.buffer,
+                                          buffer.length);
+    #else
+            store_cert = wolfSSL_X509_STORE_CTX_get_current_cert(store);
+    #endif
+
+    #ifdef CONFIG_WOLFSSL_DEBUG_CERT_BUNDLE
+        chain = wolfSSL_X509_STORE_CTX_get_chain(store);
+        numCerts = wolfSSL_sk_X509_num(chain);
+        if (!chain) {
+            numCerts = 0; /* Verification failed. */
+        }
+        ESP_LOGI(TAG, "Number of certificates in chain: %d", numCerts);
+        print_cert_subject_and_issuer(store);
+    #endif
+#else
+        store_cert = store->current_cert;
+#endif
+        if (store_cert == NULL) {
+            ESP_LOGE(TAG, "Failed to get current certificate.\n");
+            ret = WOLFSSL_FAILURE;
+        }
+        else {
+            ret = WOLFSSL_SUCCESS;
+        }
+    } /* this (ret == WOLFSSL_SUCCESS) step to get cert from store */
+
+
+    /* Get the target name and subject from the current_cert(store) */
+    if (ret == WOLFSSL_SUCCESS) {
+        store_cert_subject = wolfSSL_X509_get_subject_name(store_cert);
+        if (wolfSSL_X509_NAME_oneline(store_cert_subject, subjectName,
+                                      sizeof(subjectName)) == NULL) {
+            ESP_LOGE(TAG, "Error converting subject name to string.");
+            ret = WOLFSSL_FAILURE;
+        }
+        else {
+            ESP_LOGCBI(TAG, "Store Cert Subject: %s", subjectName );
+        }
+        store_cert_issuer = wolfSSL_X509_get_issuer_name(store_cert);
+        if (store_cert_issuer == NULL) {
+            ESP_LOGE(TAG, "Error converting Store Cert Issuer to string");
+            ret = WOLFSSL_FAILURE;
+        }
+        else {
+            ESP_LOGCBI(TAG, "Store Cert Issuer:  %s", store_cert_issuer->name );
+        }
+    }
+
+    /* When the server presents its certificate, the client checks if this
+     * certificate can be traced back to one of the CA certificates in the
+     * bundle.
+     *
+     * NOTE: To save memory, the store `cert` from above is overwritten below.
+     * Any details needed from the store `cert` should have been saved.
+     *
+     * We'll proceed by assiging `cert` to each of the respective items in
+     * bundle as we attempt to find the desired cert: */
+    if (ret == WOLFSSL_SUCCESS) {
+        _cert_bundled_loaded = 1;
```

Comment:
I see where this is set but not where it is used.

Reply:
This is intended for end users. Added  `wolfssl_cert_bundle_loaded()` to file: `esp_crt_bundle.h` .

Also renamed removed stray middle 'd' from `_cert_bundled_loaded `.